scripts/generate_docs.py

```diff
@@ -213,7 +213,7 @@ def generate_ref(dstpath, ref):
     util.removePath(refpath)
     util.makePath(refpath)
 
-    util.jsonWrite(os.path.join(refpath, "level_zero.json"), ref)
```

Comment:
You are only modifying the function that writes out this json file. If there's no corresponding reader and this is unused, I suggest we just remove this whole thing.

Reply:
You are correct. This file is not used or read by anything. I'm not 100% sure why it exists, but it's places in `/docs/ref` perhaps for debugging reasons or as a reference? 
If you feel it's not worthwhile then we can remove.